LSE-61.tex

```diff
@@ -359,6 +365,28 @@ \subsubsection{Nightly Data Accessible Within 24 hrs}
 \paramname{%
 OTT1
 } \\\hline
+Fraction of detectable alerts for which an alert is actually transmitted within latency \textbf{OTT1} (see LSR-REQ-0101).
+&
+98
+&
+\unitname{%
+percent
+}
+&
+\paramname{%
+OTR1
+} \\\hline
+The instantaneous peak number of alerts per standard visit.
```

Comment:
"Instantaneous" is not an ideal word here.  OTR1 is not a rate but a quantity.

Reply:
Hmm, I do think of it as a rate: number per visit (number of alerts per 35 seconds). The term "instantaneous" is useful to differentiate nAlertVisitAvg from nAlertVisitPeak. Perhaps "The instantaneous peak rate of alerts per standard visit" is a better definition of nAlertVisitPeak?